cmd/livepeer_cli/wizard_stats.go

```diff
@@ -325,14 +315,6 @@ func (w *wizard) getEthBalance() string {
 	return e
 }
 
-func (w *wizard) getDeposit() string {
-	e := httpGet(fmt.Sprintf("http://%v:%v/broadcasterDeposit", w.host, w.httpPort))
```

Comment:
Don't we still need a deposit?

Reply:
Yeah, but it is with a different contract so this PR cleaned up the old code and the new code for the new contract is included in #658 which fetches all of a PM sender's information in a single request instead of individually fetching the sender's deposit and penalty escrow.